### PR TITLE
Add 2020-10-12-ambrosia-alpha-release

### PR DESCRIPTION
**Pull request by D4nte**

### CODE REVIEW

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# A Change of Direction
+
+Last year, we spent time and effort creating tools, such as [`create-comit-app` and `comit-scripts`](https://github.com/comit-network/create-comit-app/) to allow developers to build their own COMIT Apps.
+We organised local meetups in Sydney and even a hackathon in Singapore.
+We had a number of developers excited about the technology and possibilities that COMIT bring in but very few committed to build an app.
+
+We know realize that before other build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
```
Comment (da-kami) [inline blog/2020-10-12-ambrosia-alpha-release.md:17]:
```suggestion
We now realize that before other build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+# Ambrosia
+
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-10-12/ambrosia.png')} />
+
+This is why we spent the last few months working on [Ambrosia](https://github.com/comit-network/ambrosia/), an UI for trading in the COMIT network.
+Ambrosia allows you to trade BTC/DAI in a decentralised manner. No third party to trust, no central orderbook, no KYC.
+It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) has a bakend.
```
Comment (da-kami) [inline blog/2020-10-12-ambrosia-alpha-release.md:25]:
```suggestion
It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) as backend.
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Not only you do not need to trust a third party with your fund when trading, even our software is non-custodial.
+Ambrosia never holds your crypto, it interfaces with your hardware wallet (nano ledger s only) to make payments.
+
+We believe that custody and security are important criteria to our users, hence we also assume that users are running their own [Bitcoin Core](https://github.com/bitcoin/bitcoin) node.
+
+## Trading pair
+
+The COMIT Network Daemon (cnd) supports various trading pairs: BTC/ETH, BTC/ERC20, Lightning BTC/ETH, etc.
+However, we have limited Ambrosia to the BTC/DAI pair as we know that Bitcoin/USD.. are usually the most popular pair.
```
Comment (da-kami) [inline blog/2020-10-12-ambrosia-alpha-release.md:39]:
```suggestion
However, we have limited Ambrosia to the BTC/DAI pair as we know that Bitcoin/USD... is the most popular pair.
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+## Decentralised Orderbook
+
+At this stage, Ambrosia can only take offers.
```
Comment (da-kami) [inline blog/2020-10-12-ambrosia-alpha-release.md:43]: ? This might be confusing to people. 
Ambrosia can actually create arbitrary orders, but because there is nobody in the market yet (besides our nectar) it is very likely that an order that is not in nectar's favor won't match.

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+We have a [nectar](https://github.com/comit-network/comit-rs) instance that runs and acts as a maker.
+
+## Trying it out
+
+The software is currently in an alpha state, bugs are known and expected.
+
+If you want to try out, we strongly recommend you have a chat with us so we can assist:
```
Comment (da-kami) [inline blog/2020-10-12-ambrosia-alpha-release.md:50]:
```suggestion
If you want to try Ambrosia out, we strongly recommend you have a chat with us so we can assist:
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+-   Email: [team@coblox.tech](mailto:team@coblox.tech)
+-   Matrix chat: [#comit:matrix.org](https://matrix.to/#/!HYBOPcopXgKbEnEELc:matrix.org?via=matrix.org&via=privacytools.io) (e2e encryption enabled)
+-   Twitter: [@comit_network](https://twitter.com/comit_network)
+
+If you want to try it yourself, you need 2 components:
```
Comment (da-kami) [inline blog/2020-10-12-ambrosia-alpha-release.md:56]: If you add this section you might also want to add that Ambrosia depends on Ledger Nano S being present. 
And the full-node requirement.

**Review by da-kami — COMMENTED** [review]
Few fixes, will have another look once Netlify passes.

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 1 above)
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:17]:
```suggestion
We now realized that before we can expect others build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+# A Change of Direction
+
+Last year, we spent time and effort creating tools, such as [`create-comit-app` and `comit-scripts`](https://github.com/comit-network/create-comit-app/) to allow developers to build their own COMIT Apps.
+We organised local meetups in Sydney and even a hackathon in Singapore.
+We had a number of developers excited about the technology and possibilities that COMIT bring in but very few committed to build an app.
+
+We know realize that before other build successful apps with our tools, we should demonstrate what can be done with COMIT ourselves.
+
+# Ambrosia
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:19]:
```suggestion
# Introducing Ambrosia
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-10-12/ambrosia.png')} />
+
+This is why we spent the last few months working on [Ambrosia](https://github.com/comit-network/ambrosia/), an UI for trading in the COMIT network.
+Ambrosia allows you to trade BTC/DAI in a decentralised manner. No third party to trust, no central orderbook, no KYC.
+It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) has a bakend.
+
+The software has some rough edges, but we wanted to get it out there to have your feedback. Here are some choices we want your input on:
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:27]:
```suggestion
The software has some rough edges, but we wanted to get it out there to have your feedback.
Here are some choices we want your input on:
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+## Custody
+
+Not only you do not need to trust a third party with your fund when trading, even our software is non-custodial.
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:31]:
```suggestion
Not only do you not need to trust a third party with your funds when trading, even our software is completely non-custodial.
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+Ambrosia never holds your crypto, it interfaces with your hardware wallet (nano ledger s only) to make payments.
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:32]:
```suggestion
Ambrosia never holds your crypto, instead it interfaces directly with your hardware wallet (nano ledger s only) to make transactions.
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+We believe that custody and security are important criteria to our users, hence we also assume that users are running their own [Bitcoin Core](https://github.com/bitcoin/bitcoin) node.
+
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:35]: If your hardware is beefy enough, you can even run your own Ethereum node :)
But for those who don't want that, we made sure it works with Infura!

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+## Trading pair
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:36]: I would not mention that, it makes it sound less good :D

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+
+The COMIT Network Daemon (cnd) supports various trading pairs: BTC/ETH, BTC/ERC20, Lightning BTC/ETH, etc.
+However, we have limited Ambrosia to the BTC/DAI pair as we know that Bitcoin/USD.. are usually the most popular pair.
+
+## Decentralised Orderbook
+
+At this stage, Ambrosia can only take offers.
+We have a [nectar](https://github.com/comit-network/comit-rs) instance that runs and acts as a maker.
```
Comment (thomaseizinger) [inline blog/2020-10-12-ambrosia-alpha-release.md:44]: Do we want to mention that? :D

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 4 above)
Comment (thomaseizinger) [reply blog/2020-10-12-ambrosia-alpha-release.md:43]: Agree that we should rephrase that.

**Review by thomaseizinger — COMMENTED** [review]
Nice work, thanks for writing this!

I really like the intro :)
Noted down a few points!

**Review by thomaseizinger — COMMENTED** [review]
We should have a divider somewhere after the first paragraph so that it only renders a preview in the overview: https://deploy-preview-88--comit-network.netlify.app/blog/

**Comment by da-kami** [comment]
I'd use a more expressive screenshot, e.g.:

Screen Shot 2020-10-13 at 3 35 54 pm

**Comment by da-kami** [comment]
> We should have a divider somewhere after the first paragraph so that it only renders a preview in the overview: https://deploy-preview-88--comit-network.netlify.app/blog/

You can achieve that with `<!--truncate-->`

**Comment by da-kami** [comment]
I'd mention more visibly that you swap on `mainnet` - it is not mentioned in the post. What do the others think?

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 6 above)
Comment (D4nte) [reply blog/2020-10-12-ambrosia-alpha-release.md:56]: Yeah, I hesitated, will add.

**Comment by D4nte** [comment]
> I'd use a more expressive screenshot, e.g.:
> 
> Screen Shot 2020-10-13 at 3 35 54 pm
> 
> I am happy to provide one if needed :)

Please do

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 12 above)
Comment (D4nte) [reply blog/2020-10-12-ambrosia-alpha-release.md:37]: what do you mean?

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 13 above)
Comment (D4nte) [reply blog/2020-10-12-ambrosia-alpha-release.md:45]: yeah in case one try to swap with another ambrosia

**Comment by D4nte** [comment]
> I really like the intro :)

Thanks, I really wasn't sure how to tackle this.

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+Ambrosia allows you to trade BTC/DAI in a decentralised manner. No third party to trust, no central orderbook, no KYC.
+It uses [cnd](https://github.com/comit-network/ambrosia/comit-rs) as backend.
+
+The software has some rough edges, but we wanted to get it out there to have your feedback.
+Here are some choices we want your input on:
+
+## Custody
+
+Not only do you not need to trust a third party with your funds when trading, even our software is completely non-custodial.
+Ambrosia never holds your crypto, instead it interfaces directly with your hardware wallet (nano ledger s only) to make transactions.
```
Comment (bonomat) [inline blog/2020-10-12-ambrosia-alpha-release.md:35]: I propose to rephrase this as I had to read the first sentence twice to understand it: 
```suggestion
Ambrosia never holds your crypto, instead it interfaces directly with your hardware wallet (nano ledger s only) to make transactions. 
This is a huge step up towards trustless trading compared to traditional centralized solutions where you have to deposit your funds into an exchange first. 
```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+
```
Comment (bonomat) [inline blog/2020-10-12-ambrosia-alpha-release.md:36]: Not sure how to do quotes in markdown anymore, but this could fit here :) 
```suggestion

> Not your keys not your crypto

```

File: blog/2020-10-12-ambrosia-alpha-release.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+The software is currently in an alpha state, bugs are known and expected.
+We are using on mainnet and invite you to try it too.
+
+We do not provide a testnet setup simply because it can be cumbersome to find working faucets or use an hardware wallet.
+Also, we have noticed that the experience can differ on mainnet and testnet.
+
+If you want to try out, we strongly recommend you have a chat with us so we can assist:
+
+-   Email: [team@coblox.tech](mailto:team@coblox.tech)
```
Comment (bonomat) [inline blog/2020-10-12-ambrosia-alpha-release.md:62]: 🚧 This email should be internal only! 
```suggestion
-   Email: [hello@comit.network](mailto:hello@comit.network)
```

**Review by bonomat — APPROVED** [review]
Good write up. Please fix the contact email: `team@coblox.tech` is only reachable from within coblox.

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 12 above)
Comment (thomaseizinger) [reply blog/2020-10-12-ambrosia-alpha-release.md:40]: That we only support one trading pair on the frontend but more on the backend. Sounds like it is a technical design choice that is quite irrelevant to the user :)

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 13 above)
Comment (thomaseizinger) [reply blog/2020-10-12-ambrosia-alpha-release.md:47]: Well, what about the "don't mention publicly that we are running a nectar instance"? I think @bonomat had concerns here :man_shrugging:

Comment (bonomat) [reply blog/2020-10-12-ambrosia-alpha-release.md:47]: Good point. Please change it to show that this is only for test purposes :) 
```suggestion
For test purposes only we have a temporary [nectar](https://github.com/comit-network/comit-rs) instance that runs and acts as a maker.
```

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 12 above)
Comment (D4nte) [reply blog/2020-10-12-ambrosia-alpha-release.md:40]: It is supposed to sound like that we could support any pair if people ask.

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 15 above)
Comment (D4nte) [reply blog/2020-10-12-ambrosia-alpha-release.md:36]: You are making my life hard by doing 2 different suggestions on the same piece of text. Please don't.

File: blog/2020-10-12-ambrosia-alpha-release.md (hunk 12 above)
Comment (D4nte) [reply blog/2020-10-12-ambrosia-alpha-release.md:42]: corrected.
